tests/sentry/api/endpoints/test_sentry_internal_app_tokens.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import
+
+from django.core.urlresolvers import reverse
+
+from sentry.testutils import APITestCase
+from sentry.testutils.helpers import with_feature
+from sentry.models import ApiToken
+
+
+class SentryInternalAppTokenCreationTest(APITestCase):
+    def setUp(self):
+        self.user = self.create_user(email='boop@example.com')
+        self.org = self.create_organization(owner=self.user, name='My Org')
+        self.project = self.create_project(organization=self.org)
+
+        self.internal_sentry_app = self.create_internal_integration(
+            name='My Internal App',
+            organization=self.org
+        )
+
+        self.url = reverse(
+            'sentry-api-0-sentry-internal-app-tokens',
+            args=[self.internal_sentry_app.slug],
+        )
+
+    @with_feature('organizations:sentry-apps')
+    def test_create_token(self):
+        self.login_as(user=self.user)
+        response = self.client.post(self.url, format='json')
+        assert response.status_code == 201
+
+        assert ApiToken.objects.get(token=response.data['token'])
+
+    @with_feature('organizations:sentry-apps')
+    def test_non_internal_app(self):
```

Comment:
Should you have tests for the not found scenarios too?

Reply:
@markstory Added the case where the slug is for an app that doesn't exist. Is that what you had in mind?

Reply:
Yes 🙇